core/domain/state_domain.py

```diff
@@ -3076,35 +3076,48 @@ def _get_all_translatable_content(self):
         translatable through the contributor dashboard.
 
         Returns:
-            dict(str, str). Returns a dict with key as content id and content
-            html/unicode as the value.
+            dict(str, WrittenTranslation). Returns a dict with key as content id
```

Comment:
Aren't we misusing the `WrittenTranslation` domain object here? It's counter-intuitive to me to use the `translation` field of a `WrittenTranslation` to store the *untranslated* content text. 

It seems to me we want to return some object with the following data:
* data format
* untranslated content text

Maybe we need a new domain object for this?

Reply:
Makes sense! I can create a new domain object. Any suggestions on how to name this? I'm thinking `UntranslatedContent` but not sure.

Reply:
I think with the current conventions in the codebase, `TranslatableContent` would make sense, e.g. we already have data formats like 'TranslatableSetOfNormalizedString'.

And I realize this change will need a lot of find and replacing so thanks in advance!

Reply:
Yep, that makes sense. Thanks! 

And yes it does involve changing in a bunch of places haha. But I think it might be worth it!

Reply:
Done.

Reply:
Per [design doc](https://docs.google.com/document/d/1gSrPAuccwmfWdGvygVNFS6pEPdn7fm8n0p4ZKhX1R-w/edit#), a better name would be `TranslatableItem`. I have updated it accordingly in my PR. Thanks!